packages/react-divider/src/components/Divider/Divider.types.ts

```diff
@@ -1,48 +1,42 @@
-import * as React from 'react';
-import { ComponentPropsCompat, ShorthandPropsCompat } from '@fluentui/react-utilities';
+import { ComponentProps, ComponentState, IntrinsicShorthandProps } from '@fluentui/react-utilities';
 
-export type DividerProps = ComponentPropsCompat &
-  React.HTMLAttributes<HTMLElement> & {
-    /**
-     * Determines the alignment of the content within the divider.
-     * @defaultvalue 'center'
-     */
-    alignContent?: 'start' | 'end' | 'center';
-
-    /**
-     * A divider can have one of the preset appearances.
-     * When not specified, the divider has its default appearance.
-     */
-    appearance?: 'brand' | 'strong' | 'subtle';
-
-    /**
-     * Adds padding to the begining and end of the divider
-     */
-    inset?: boolean;
+export type DividerSlots = {
+  root: IntrinsicShorthandProps<'div'>;
+  /**
```

Comment:
nit: Add a new line here
```suggestion
  root: IntrinsicShorthandProps<'div'>;
  
  /**
```